ykrt/src/compile/jitc_yk/jit_ir.rs

```diff
@@ -1171,6 +1421,16 @@ impl Module {
         }
     }
 
+    /// Return the const for the specified index.
+    ///
+    /// # Panics
+    ///
+    /// Panics if the index is out of bounds.
+    pub(crate) fn const_<'a>(&'a self, idx: ConstIdx) -> &'a Constant {
```

Comment:
I added this function in https://github.com/ykjit/yk/pull/1029/files#diff-d66ba8eec98a7d4ee4f268ff4459098b17150480892e69b748f2b50dfd5d2cb9R1072, so this can probably go once you rebase.

Reply:
I added a merge commit and inherited it.

Reply:
This is still here. I expected this to disappear after the rebase.

Reply:
Oops. Fixed in d0529708